pip_requirements.txt

```diff
@@ -1,8 +1,20 @@
-click
+IPython < 4.0.0
```

Comment:
@jwdebelius, we currently have IPython < 4.0.0 in our conda requirements, but iirc, you've been operating without issue on IPython > 4.0.0, correct?